Instructions.md

```diff
@@ -0,0 +1,32 @@
+# Instructions for testing RBAC support for Linux in .NET 5 preview 6
+
+1. Install .NET 5 preview 6 SDK
```

Comment:
Not all builds are coherent, link to specific builds that you tested.

Reply:
Added a specific sdk build.